hook.yaml

```diff
@@ -74,6 +74,13 @@ services:
 files:
   - path: etc/docker/daemon.json
     contents: '{"debug": true}'
+  - path: etc/profile.d/local.sh
+    contents: |
+      alias l='ls -lF'
```

Comment:
These first two aliases feel a bit like scope creep to me. Would it be OK to remove them?